fuel/src/main/kotlin/com/github/kittinunf/fuel/toolbox/HttpClient.kt

```diff
@@ -141,39 +145,47 @@ internal class HttpClient(
             headers.remove(Headers.CONTENT_LENGTH)
             contentLength = -1
         }
-
+        val inputStream = stethoHook?.interpretResponseStream(connection.inputStream)
```

Comment:
I prefer something along the lines of `wrapResponseStream` as it's more clear what the intent is of the function